service/scaladsl/api/src/main/scala/com/lightbend/lagom/scaladsl/api/transport/HeaderFilter.scala

```diff
@@ -119,20 +119,34 @@ object HeaderFilter {
  * If using this on a service that serves requests from the outside world, it would be a good idea to block the
  * `User-Agent` header in the web facing load balancer/proxy.
  */
+//#user-agent-header-filter
```

Comment:
The changes to this file are just formatting to ensure that it looks good when displayed in the documentation.